bundles/org.openhab.binding.coolmasternet/src/main/java/org/openhab/binding/coolmasternet/internal/ControllerHandler.java

```diff
@@ -146,9 +151,10 @@ public boolean isConnected() {
                 logger.trace("Sending command '{}'", command);
                 OutputStream out = localSocket.getOutputStream();
                 out.write(command.getBytes());
-                out.write("\r\n".getBytes());
+                out.write("\n".getBytes());
 
-                try (Reader isr = new InputStreamReader(localSocket.getInputStream());
+                try (InputStream uis = new UncloseableInputStream(localSocket.getInputStream());
```

Comment:
If the stream isn't meant to be closed, there is no reason to use a try-with-resources.